packages/cheqd/package.json

```diff
@@ -26,7 +26,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "@cheqd/sdk": "^2.5.1",
+    "@cheqd/sdk": "^2.6.0",
```

Comment:
We should properly test this in RN. We have to pin everything to a very specific version or the SDK breaks. Even with the current version it throws an unhandled promise rejection on startup (due to wasm errors) but it somehow still works for resolving.

We haven't had the time to properly look into it, but before we can merge this we should first test it in react native (actually we should pin the version to 2.5.1 for now, instead of using ^2.5.1 until it's fixed)

Reply:
Reverting it! It would be very helpful if you can create an issue on our [sdk](https://github.com/cheqd/sdk) with context-rich logs or reproduction steps. Thanks

Reply:
@TimoGlastra after our mainnet upgrade we got some feedback that Cheqd transactions from credo are failing. This must be due to insufficient gas after introducing our fee market module, In our latest sdk we bumped the default gas fees, I think we should bump it. We have not made any major dependency changes from 2.5.1 to 2.6.0